.vib/grafana-loki/goss/goss.yaml

```diff
@@ -0,0 +1,25 @@
+file:
```

Comment:
I may be missing something but why are we checking only NGINX and its configuration? Aren't there any other tests related to grafana-loki itself that can be performed in any of the deployed pods? IIRC, there should be both a `loki` and `logcli` binaries in the pods.

Reply:
We are testing the API, which is served through `NGINX`. My point is that if Cypress tests succeed, that would imply that the API was responsive and it is highly likely that NGINX is properly configured then.

I don't have the knowledge background about Grafana-Locki, but maybe there are other pods to test that may be more interesting than the gateway itself (i.e. the suggested `loki` & `logcli` binaries). Else, there is no problem in testing NGINX here.

Reply:
I've added 2 binary tests in a separate file, testing a separate pod. 
Please let me know if it's OK for you.

Reply:
I think this is a good solution! No issues from my side 👍